styles/button/button.css

```diff
@@ -0,0 +1,86 @@
+@import url("btnhover.css");
+
+.button{
+    background-color: var(--default-color);
+    color: white;
+    min-width: 100px;
+    width: max-content;
+    padding: 0.8rem 1rem;
+    border-radius: 1px;
+    /* border: none; */
+    cursor: pointer; 
+    /* transition-property: all; */
```

Comment:
whenever you feel comfortable remove these unwanted comments

Reply:
Noted, done.